specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/capacity.json

```diff
@@ -0,0 +1,180 @@
+        },
+        "tags": [
+          "ContainerRegistryCapacity"
+        ],
+        "description": "Returns container registry capacity property.",
+        "operationId": "ContainerRegistryCapacity_Get",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CapacityNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Container registry capacity property is being returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryCapacity"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/capacity": {
+      "get": {
+        "x-ms-examples": {
+          "Returns container registry capacity properties.": {
+            "$ref": "./examples/Configuration/ListCapacity.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryCapacities"
+        ],
+        "description": "Returns a list of container registry capacity properties.",
+        "operationId": "ContainerRegistryCapacity_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Container registry capacity properties are being returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryCapacityList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryCapacity": {
+      "description": "Container registry capacity property.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry capacity property.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryCapacityProperty"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
```

Comment:
use the common proxy resourcer type for this. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json

Reply:
We don't have access to the types from our repo:
https://github.com/msazurestackworkloads/azure-rest-api-specs

Reply:
same comments